src/assets/css/index.css

```diff
@@ -0,0 +1,529 @@
+	z-index: 100;
+	border-top-right-radius: 50%;
+	border-top-left-radius: 50%;
+	border-bottom-right-radius: 50%;
+	border-bottom-left-radius: 50%;
+}
+.second-item {
+	float: right;
+	align-content: center;
+	justify-content: center;
+}
+.second-item img {
+	height: 37rem;
+	width: 45rem;
+	padding-bottom:1.5rem;
+}
+
+.header-title {
+	height: 12rem;
+	width: 100%;
+	background-color: #282828;
+	position: fixed;
+	z-index: 101;
+}
+.header-title .top-title.wrapper {
+	display: flex;
+	align-items: center;
+}
+.header-title .header_img {
+	width: 39.2rem;
+	height: 8.8rem;
+}
+
+.header-bar {
+	min-height: 63rem;
+	width: 100%;
+	background-image: url('../img/header-img.png');
+	background-repeat: no-repeat;
+	background-size: cover;
+	background-position: 40% 50%;
+	image-rendering: pixelated;
+	margin-top: -1px;
+	padding-top: 12rem;
+	padding-bottom: 1rem;
+}
+.header-bar .title {
+	font-weight: bold;
+	color: #a97b3f;
+}
+.header-bar .sub-title {
+	color: white;
+	max-width: 120rem;
+	padding-top: 2.7rem;
+}
+.header-bar p {
+	padding-top: 3.2rem;
+	max-width: 64.3rem;
+	color: white;
+	font-size: 2.8rem;
+}
+.header-bar .header.wrapper {
+	padding-top: 9.3rem;
+}
+.header-bar .header-button {
+	background-color: #a97b3f;
+	color: white;
+	padding: 2.2rem 4rem;
+	border-radius: 4rem;
+	border: none;
+	font-size: 2.4rem;
+}
+.header-button:hover {
+	cursor: pointer;
+	background-color: #815d2e;
+	-moz-transition: all 0.1s ease-in;
+	-webkit-transition: all 0.1s ease-in;
+	-o-transition: all 0.1s ease-in;
+	transition: all 0.1s ease-in;
+}
+.coin-works {
+	text-align: center;
+	width: 100%;
+}
+.coin-works .row-item:nth-child(2) {
+	margin: 0 5rem;
+}
+.coin-use {
+	text-align: center;
+	background-color: #d8d8d8;
+	width: 100%;
+}
+.coin-use .row-item:nth-child(2) {
+	margin: 0 5rem;
+}
+.coin-use .sub-title {
+	padding-top: 6rem;
+}
+.coin-buy .row {
+	height: 100%;
+	justify-content: space-between;
+}
+.coin-buy .sub-title {
+	padding-top: 6rem;
+}
+.coin-buy .description {
+	padding-top: 3.5rem;
+	padding-bottom: 5.5rem;
+	width: 43rem;
+}
+.coin-buy .step-1 {
+	margin-bottom: 7.2rem;
+}
+.coin-buy .step-2 {
+	margin-bottom: 4.5rem;
+}
+.coin-buy .first-item .sub-title {
+	max-width: 68rem;
+}
+.coin-redeem .row {
+	height: 100%;
+}
+.coin-redeem .sub-title {
+	padding-top: 6.1rem;
+}
+.coin-redeem .description {
+	padding-top: 3.5rem;
+	padding-bottom: 5.5rem;
+}
+.coin-redeem .second-item {
+	padding-right: 23rem;
+}
+.coin-redeem .step-1 {
+	margin-bottom: 4.5rem;
+}
+.coin-redeem .step-2 {
+	margin-bottom:4.2rem;
+}
+.technical-implementation {
+	text-align: center;
+	width: 100%;
+}
+.technical-implementation .sub-title {
+	padding-top: 6rem;
+}
+.technical-implementation .row-item:nth-child(2) {
+	margin: 0 5rem;
+}
+.faq {
+	text-align: center;
+	width: 100%;
+}
+.faq .sub-title {
+	padding-top: 6rem;
+}
+.faq-section {
+	text-align: left;
+}
+.question {
+	padding-top: 3rem;
+	font-weight: bold;
+	font-size: 2.4rem;
```

Comment:
it's better to style the h4 element itself rather than add specific font-size to a class, that way it can be used throughout and removes the need for repetition of font-size etc. for example a padding-top class can be added to be used throughout, and instead of adding styling for bold font you can use the `<strong>` tag where needed. it doesn't need to be so precise, padding classes can be generalized and used for different elements so the css can become much shorter. if you can combine certain styles under the same element instead of many classes it will make the code easier to manage and future sections to be added more easily.